Composer/packages/client/src/constants.ts

```diff
@@ -399,25 +399,22 @@ export const triggerNotSupportedWarning = () =>
     'This trigger type is not supported by the RegEx recognizer. To ensure this trigger is fired, change the recognizer type.'
   );
 
-export const feedDictionary: { [key in FeedName]: string } = {
-  firstPartyCsharp:
-    'https://registry.npmjs.org/-/v1/search?text=conversationalcore+scope:microsoft&size=100&from=0&quality=0.65&popularity=0.98&maintenance=0.5',
-  firstPartyNode: '',
-};
+export const firstPartyTemplateFeed =
+  'https://registry.npmjs.org/-/v1/search?text=generator+keywords:bf-template+scope:microsoft'; // +maintainer:botframework
```

Comment:
Is there a reason the extra query parameters (like quality and popularity) are going away here?

Reply:
They were unneeded, we want to ensure that every generator in npm within the Microsoft scope, with the bf-template keyword is returned by this feedUrl. Its on our backlog to add the Maintainer:botframework to this search query to enforce a stricter contract. These at the only qualifiers we need for the template feed so things like quality and popularity are not valid qualifiers for our use case.